slicetag.sh

```diff
@@ -48,7 +48,7 @@ set -e
 command=$1
 shift || :
 
-TAG=$( git rev-list --all | wc -l ) 
+TAG=$( date -u +"%s" ) 
```

Comment:
No, it's not unique to the timestamp approach. Resolving this issue was one of the main intentions I had in using the current timestamp... that all machines (assumption of working ntp) would be generating the release number from the same source (the current time), no matter which fork or branch or machine or commit count, etc. I suppose one solution to this would be to just standardize on which repo builds are made from.  Perhaps we need to have several branches master and testing, or something like that, and all builds are always done from the m-lab/ndt-support repo on one of those branches.